_src/recipes/010-add-a-sitemap.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Add a Sitemap to 11ty site
+authorName: Akbarsait
+authorUrl: https://www.akbarsait.com
+---
+
+Sitemap helps search engine crawl for making easier and quicker way to find and index your website content. The base blog of 11ty is already equipped but here we have added few changes especially for the dates to make it easier and readable. 
+
+### Directions
+1. Create a file in your source directory  called **sitempa.njk** and add the following content to it.
+
+```md
+---
+permalink: /sitemap.xml
+eleventyExcludeFromCollections: true
+---
+<?xml version="1.0" encoding="utf-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    {% for page in collections.all %}
+        <url>
+            <loc>{{ site.url }}{{ page.url | url }}</loc>
```

Comment:
This assumes the user has a `data.json` file, which we don't have a recipe for yet. You can either make a recipe and add it to the top as a prerequisite, or just note which values need to exist in data.json for this to work.

Reply:
I have updated and pushed the changes now.